boards/arm/mimxrt1064_evk/Kconfig.defconfig

```diff
@@ -10,6 +10,14 @@ if BOARD_MIMXRT1064_EVK
 config BOARD
 	default "mimxrt1064_evk"
 
+choice CODE_LOCATION
+        default CODE_FLEXSPI2
```

Comment:
How are you programming the code to flash? Last I checked, JLink didn't support this yet.

@jhqian Did Segger resolve this issue recently?

Reply:
> How are you programming the code to flash? Last I checked, JLink didn't support this yet.

I use drag-and-drop method for now.

Reply:
Let's keep the defaults (both code and data) in TCM until the tools can properly support flash programming.